flang/include/flang/Support/Fortran-features.h

```diff
@@ -79,7 +79,7 @@ ENUM_CLASS(UsageWarning, Portability, PointerToUndefinable,
     CompatibleDeclarationsFromDistinctModules, ConstantIsContiguous,
     NullActualForDefaultIntentAllocatable, UseAssociationIntoSameNameSubprogram,
     HostAssociatedIntentOutInSpecExpr, NonVolatilePointerToVolatile,
-    RealConstantWidening, VolatileOrAsynchronousTemporary)
+    RealConstantWidening, VolatileOrAsynchronousTemporary, BadValueInDeadCode)
```

Comment:
This could be `WarnInDeadCode` if we think we will want one flag to control all warnings in dead code. Otherwise this seems like an ok name for issues at hand.

Reply:
`UsageWarning::WarnInDeadCode` seems redundant to me.